### ISSUE TITLE
Clicking on the current date in the navigation area causes a crash. 

### ISSUE THREAD

**Issue by fiveNinePlusR** [jbkuczma/NBAreact#71]
date.split('/') is undefined. 

```
TypeError: date.split is not a function

This error is located at:
    in ScoresScreen (created by Connect(ScoresScreen))
    in Connect(ScoresScreen) (at SceneView.js:17)
    in SceneView (at CardStack.js:385)
    in RCTView (at View.js:71)
    in View (at createAnimatedComponent.js:147)
    in AnimatedComponent (at Card.js:12)
    in Card (at PointerEventsContainer.js:39)
    in Container (at CardStack.js:414)
    in RCTView (at View.js:71)
    in View (at CardStack.js:348)
    in RCTView (at View.js:71)
    in View (at CardStack.js:347)
    in CardStack (at CardStackTransitioner.js:66)
    in RCTView (at View.js:71)
    in View (at Transitioner.js:142)
    in Transitioner (at CardStackTransitioner.js:19)
    in CardStackTransitioner (at StackNavigator.js:38)
    in Unknown (at createNavigator.js:13)
    in Navigator (at createNavigationContainer.js:214)
    in NavigationContainer (at SceneView.js:17)
    in SceneView (at TabView.js:30)
    in RCTView (at View.js:71)
    in View (at TabView.js:29)
    in RCTView (at View.js:71)
    in View (at TabViewPagerPan.js:251)
    in RCTView (at View.js:71)
    in View (at createAnimatedComponent.js:147)
    in AnimatedComponent (at TabViewPagerPan.js:238)
    in TabViewPagerPan (at TabView.js:117)
    in RCTView (at View.js:71)
    in View (at TabViewAnimated.js:194)
    in TabViewAnimated (at TabView.js:178)
    in TabView (at withCachedChildNavigation.js:44)
    in withCachedChildNavigation(TabView) (at TabNavigator.js:32)
    in Unknown (at createNavigator.js:13)
    in Navigator (at createNavigationContainer.js:214)
    in NavigationContainer (at App.js:29)
    in Provider (at App.js:28)
    in App (at renderApplication.js:35)
    in RCTView (at View.js:71)
    in View (at AppContainer.js:102)
    in RCTView (at View.js:71)
    in View (at AppContainer.js:122)
    in AppContainer (at renderApplication.js:34)
handleException @ ExceptionsManager.js:65
handleError @ InitializeCore.js:115
reportFatalError @ error-guard.js:44
__guardSafe @ MessageQueue.js:318
callFunctionReturnFlushedQueue @ MessageQueue.js:117
(anonymous) @ debuggerWorker.js:72
```

**Comment by jbkuczma** [comment]
Thanks for posting this bug. I'll be pushing out a release that fixes this. In the update, when you click on the date, it will take you back to the games for todays date.

**Comment by jbkuczma** [comment]
Possibly in the future (not a high priority for me at the moment), clicking on the date would show a calendar that you can easily pick a date from. Thoughts?

**Comment by fiveNinePlusR** [comment]
The calendar thing could be pretty nice to have but maybe more like a week view instead of a month view? or maybe an infinite vertical scroll that lists games in order?

**Comment by jbkuczma** [comment]
Problem has been fixed in v2.3 and merged to master/released.
